packages/protocol-http/src/httpResponse.ts

```diff
@@ -1,24 +1,88 @@
-import { HeaderBag, HttpMessage, HttpResponse as IHttpResponse } from "@aws-sdk/types";
+import { HeaderBag, HttpMessage, Response } from "@aws-sdk/types";
+
+import { Fields } from "./Fields";
+import { getHeadersProxy, headersToFields } from "./headersProxy";
 
 type HttpResponseOptions = Partial<HttpMessage> & {
   statusCode: number;
 };
 
-export interface HttpResponse extends IHttpResponse {}
+type HttpResponseFromOptions = {
```

Comment:
this looks like it's part of the public API, so it should be exported, along with `HttpResponseOptions`

Reply:
Updated